tests/test_pro_document_quality.py

```
@@ -0,0 +1,247 @@
+    """Экономия сохраняется там, где она ничего не портит."""
+    for schema in (
+        "korgan_verified_legal_research",
+        "korgan_court_ready_validation",
+        "korgan_claim_v2_facts",
+        "korgan_document_extract",
+    ):
+        assert reasoning_for(schema, "gpt-5.1") == {"effort": "none"}, schema
+
+
+def test_reasoning_is_gpt51_only():
+    """Параметр reasoning не отправляется моделям, которые его не принимают."""
```

Ruff (0.16.2)

[warning] Docstring contains ambiguous `е` (CYRILLIC SMALL LETTER IE). Did you mean `e` (LATIN SMALL LETTER E)?

(RUF002)